Sources/swift-openapi-generator/runGenerator.swift

```diff
@@ -94,18 +105,31 @@ extension _Tool {
     /// - Throws: When writing to disk fails.
     /// - Returns: `true` if the generated contents changed, otherwise `false`.
     @discardableResult
-    static func replaceFileContents(at path: URL, with contents: () throws -> Data) throws -> Bool {
+    static func replaceFileContents(
+        inDirectory outputDirectory: URL,
+        fileName: String,
+        with contents: () throws -> Data
+    ) throws -> Bool {
+        let fm = FileManager.default
```

Comment:
```suggestion
        let fileManager = FileManager()
```

Variable naming, and it's better to create a new FileManager, as otherwise someone might have set the delegate on the shared file manager, which can lead to unexpected results.

Reply:
This would be inconsistent with the rest of the project.
Should i still go for it?
I'll rename the variable for now.